pkg/lang/ir/system.go

```diff
@@ -98,7 +98,7 @@ func (g *Graph) compileBase() llb.State {
 	var base llb.State
 	if g.CUDA == nil && g.CUDNN == nil {
 		if g.Language.Name == "r" {
-			base = llb.Image("docker.io/r-base:4.2.0")
+			base = llb.Image("docker.io/terrytangyuan/r4.2-envd:0.0.1")
```

Comment:
@gaocegege @VoVAllen Could you add me to docker.io/tensorchord?

Reply:
Sorry did not notice the comment here. I replied in the issue #459